kitsune/settings.py

```diff
@@ -459,7 +460,9 @@ def parse_conn_max_age(value):
 TEMPLATES = [
     {
         "BACKEND": "django_jinja.backend.Jinja2",
-        "DIRS": [],
+        "DIRS": [
+            path("dist"),
```

Comment:
why is this needed?

Reply:
so we can load the html files created by the webpack html plugin with `{% include ... %}`